### PR TITLE
169 filters

### PR DESCRIPTION
**Pull request by lillijo**

### CODE REVIEW

**Comment by mx-e** [comment]
Screenshot 2020-02-10 at 18 47 30

formate is not ticket by default

**Comment by lillijo** [comment]
> Screenshot 2020-02-10 at 18 47 30
> 
> formate is not ticket by default

This is on purpose as the Formate view is not yet ready, and having Formate and Zielgruppen links at the same time would be cluttering

File: src/store/reducer/reducer.js [hunk 1]
```diff
@@ -460,17 +466,40 @@ const changeCheckboxFilter = (state, action) => {
...
+      );
+    } else if (action.value === "Laborgeräte") {
+      newFilter.infrastructures.value = toggleAllOfSubset(
+        newFilter.infrastructures,
+        newFilter.highlevelFilter,
+        action.value
+      );
+    }
   }
   return {
```
Comment (mx-e) [inline src/store/reducer/reducer.js]: ain't pretty but i guess it works

File: src/components/ClusterMap/cluster-map-view.js [hunk 2]
```diff
@@ -268,7 +274,8 @@ export default class ClusterMapView extends React.Component {
         className={style.clusterMapWrapper}
         style={{
           width: this.props.width,
-          height: this.props.height
+          height: this.props.height,
+          marginTop: "10px"
         }}
       >
```
[the remark this answers is not in the document]
Comment (lillijo) [reply src/components/ClusterMap/cluster-map-view.js]: you are right, this was actually here before, changing values to often made me forget that

File: src/components/ClusterMap/cluster-map.js [hunk 3]
```diff
@@ -81,15 +91,24 @@ const mapStateToProps = state => {
...
+    categoriesForView = categories
+      .filter(c => c.count > 0 && filters.targetgroups.value.includes(c.title))
+      .map(
+        cat =>
+          (cat = {
+            ...cat,
+            connections: cat.connections.filter(con =>
+              filteredProjects.find(p => p.id === con.id)
+            )
+          })
```
[the remark this answers is not in the document]
Comment (lillijo) [reply src/components/ClusterMap/cluster-map.js]: i moved it to the reducer

**Pull request merged**